Clarify Readme steps to add new sample (#137)
MicrosoftEdge/Demos: pwa-installer/README.md

# PWA installer

<!-- ====================================================================== -->
## About the sample

> This demo app showcases the Web Install API.  This demo app also demonstrates CSS Masonry layout.

Reviewer: ```suggestion
This demo app showcases the Web Install API.
```

CSS Masonry was only added to make the app look nicer, it's not something that this app specifically demonstrates. Let's remove this.

**PWA installer** is a demo app for a landing page (titled **Edge demos** ) that lets you install PWAs from a collection of applications.

This PWA uses the Web Install API to install other PWAs.  Also uses CSS Masonry, as a progressive enhancement.

PWA installer uses the following features:

| Feature | Description | Documentation |
|---|---|---|
| Web Install API | PWA installer uses the `navigator.install()` API to install other PWAs on the device. | [Web Install API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/WebInstall/explainer.md) |

> | CSS Masonry | Instead of a strict grid that has gaps below shorter items, the items fill the columns in the layout to avoid gaps. | [Masonry layout](https://developer.mozilla.org/docs/Web/CSS/CSS_grid_layout/Masonry_layout) |

Reviewer: Similar to the pwa-background-sync and pwa-file-handlers samples, I think we're using too much content to say the same thing three times. How about this:

```suggestion
**PWA installer** is a sample app (titled **Edge demos** ) that lets you install other PWAs on your device.

PWA installer uses the following features:

| Feature | Description | Documentation |
|---|---|---|
| Web Install API | PWA installer uses the `navigator.install()` API to install other PWAs on the device. | [Web Install API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/WebInstall/explainer.md) |
```

<!-- ====================================================================== -->
## Open the sample

To open and run the sample:

1. Go to the [PWA installer](https://microsoftedge.github.io/Demos/pwa-installer/) demo in a new window or tab.

1. Open a new tab, and then go to `edge://flags`.

1. In the **Search** box, enter **web-app-installation-api**.

   This flag is available in browsers based on Chromium, such as Microsoft Edge, starting with version 139.

1. Set the **Web App Installation API** flag to **Enabled**.

> 1. In the **Search** box, enter **css-masonry-layout**<!-- todo: no such flag.  "CSS Grid Lanes Layout" flag exists -->.

Author: No such Edge flag, **css-masonry-layout**.  
**CSS Grid Lanes Layout** flag exists.

Delete the two css-masonry-layout steps?

https://github.com/mikehoffms/Demos/blob/user/mikehoffms/readme-tables-links/pwa-installer/README.md#open-the-sample

@captainbrosset

Reviewer: Let's remove this step, this is just a nice-to-have at this point. No need to ask users to enable the flag.

```suggestion
```

> 1. Set the **CSS Masonry Layout** flag to **Enabled**.

Reviewer: Remove this step too.

# PWA installer

<!-- ====================================================================== -->
## About the sample

This demo app showcases the Web Install API.  This demo app also demonstrates CSS Masonry layout.

**PWA installer** is a demo app for a landing page (titled **Edge demos** ) that lets you install PWAs from a collection of applications.

This PWA uses the Web Install API to install other PWAs.  Also uses CSS Masonry, as a progressive enhancement.

PWA installer uses the following features:

| Feature | Description | Documentation |
|---|---|---|
| Web Install API | PWA installer uses the `navigator.install()` API to install other PWAs on the device. | [Web Install API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/WebInstall/explainer.md) |

> | CSS Masonry | Instead of a strict grid that has gaps below shorter items, the items in the following row are raised up to fill the gaps. | [Masonry layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Masonry_layout) |

Reviewer: ```suggestion
| CSS Masonry | Instead of a strict grid that has gaps below shorter items, the items fill the columns in the layout to avoid gaps. | [Masonry layout](https://developer.mozilla.org/docs/Web/CSS/CSS_grid_layout/Masonry_layout) |
```

<!-- ====================================================================== -->
## Open the sample

To open and run the sample:

> 1. Go to [Open the demo](https://microsoftedge.github.io/Demos/pwa-installer/).

Reviewer: ```suggestion
1. Go to [PWA Installer](https://microsoftedge.github.io/Demos/pwa-installer/).
```

Author: 1. Go to the [PWA installer] ( https://microsoftedge.github.io/Demos/pwa-installer/ ) demo in a new window or tab.

   **Enable the Web Install API:**

1. Open a new tab, and then go to `about:flags`.

>    You end up at `edge://flags`.

Reviewer: ```suggestion
1. Open a new tab, and then go to `edge://flags`.
```

1. In the **Search** box, enter **web-app-installation-api**.

   This flag is available in browsers based on Chromium, such as Microsoft Edge, starting with version 139.

1. Set the **Web App Installation API** flag to **Enabled**.

1. Click the **Restart** button in the lower right.

   The browser restarts.

   **Enable CSS Masonry layout** (optional; a progressive enhancement):

1. In the browser, open a new tab and go to `about:flags`.

   You end up at `edge://flags`.

1. In the **Search** box, enter **css-masonry-layout**.

1. Set the **CSS Masonry Layout** flag to **Enabled**.

1. Click the **Restart** button in the lower right.

>    The browser restarts.

Reviewer: Can we tell users to enable both flags at the same time, while they're on the edge://flags page? Right now, we're telling them to do the steps twice in a row (open edge://flags, set a flag, restart, and then open edge://flags, set another flag, restart again)

Author: "In the **Search** box, enter **css-masonry-layout**" - no such flag.
**CSS Grid Lanes Layout** flag exists.

Author: condensed steps.  opened sep. PR Comment re: flag not found
